### PR TITLE
Added warning notes on build dependencies

### PR DESCRIPTION
**Pull request by ElleEss**
#### Description
Emergency warning about Unity build dependencies
#### Tests
Improbadoc
#### Documentation
Is docs.
#### Primary reviewers
@Helcaraxan @anne-edwards (Sam on vacay FYI only)

### CODE REVIEW

File: docs/get-started.md [hunk 1]
```diff
@@ -24,9 +24,12 @@ If you have not signed up before, you can do so [here](https://improbable.io/get
 
 - Make sure you download the **Installer** version, and select the following components during installation:
     - **Linux Build Support**
-    - **Mac Build Support**
+    - **Mac Build Support**
```
Comment (Helcaraxan) [inline docs/get-started.md]: Superfluous space

File: docs/content/troubleshooting.md [hunk 2]
```diff
@@ -20,9 +20,12 @@ When building workers (Unity Editor menu: **Build For Cloud** > **my_worker_name
  You don't have the right Unity build settings enabled.<br/>
  In your Unity Editor:
 
- * Check what build support you have enabled (menu: **File** > **Build Settings**) and make sure you have your target build platform enabled (for example, Mac if you want your game to run on Mac, Windows if you want your game to run on Windows.) 
+ * Check what build support you have enabled (menu: **File** > **Build Settings**) and make sure you have your target build platform enabled. <br/>
+You need **Mac** build support if you are developing on a Windows PC.<br/>
```
Comment (jessicafalk) [inline docs/content/troubleshooting.md]: do we want to say that? people might want to create a Windows-only game

File: docs/get-started.md [hunk 3]
```diff
@@ -24,9 +24,12 @@ If you have not signed up before, you can do so [here](https://improbable.io/get
...
 - Make sure you download the **Installer** version, and select the following components during installation:
     - **Linux Build Support**
-    - **Mac Build Support**
+    - **Mac Build Support** 
 
-> **Note:** You need Linux build support because all server-workers in a cloud deployment run in a Linux environment. 
+> **Note:** 
+You need these build supports even though you are developing on a Windows PC.<br/>
+You need **Linux Build Support** because all server-workers in a cloud deployment run in a Linux environment. <br/>
+You need **Mac Build Support** because your end-users may play your game on a Mac, so SpatialOS requires it.
```
Comment (jessicafalk) [inline docs/get-started.md]: that's not really true. they can disable building it for mac / windows if they want

File: docs/content/build.md [hunk 4]
```diff
@@ -26,7 +26,11 @@ To build the worker configs, you can either:
 
 > **It’s finished building when:** there is a message stating `'spatial build build-config' succeeded`.
 
-<%(Callout type="alert" message="If you encounter build errors, you might not have Linux build support enabled. You need Linux build support because server-workers in a cloud deployment run in a Linux environment. In the `Assets/Fps/Config/BuildConfiguration`, do not change the `UnityGameLogic Cloud Environment` from Linux. To fix this, run the Unity installer and select the build options you are missing.")%>
+<%(Callout type="alert" message="If you encounter build errors, you might not have selected the build supports your game needs during your Unity setup. <br/><br/>
+* You need **Linux** build support. This is because server-workers in a cloud deployment run in a Linux environment. In the `Assets/Fps/Config/BuildConfiguration`, do not change the `UnityGameLogic Cloud Environment` from Linux.<br/> <br/>
+* You need **Mac** build support if you are developing on a Windows PC.<br/>
+* You need **Windows** build support if you are developing on a Mac. <br/>
+This is so your game supports end-users who may be playing on a different platform to your development machine.")%>
```
Comment (anne-edwards) [inline docs/content/build.md]: Not sure if I've understood it correctly, but couldn't you simplify this to "You need Linux, Mac, and Windows build support, whatever platform you're developing on"? (This is assuming that you need the build support for the platform you're developing on, as well as the other two?)

File: docs/get-started.md [hunk 5]
```diff
@@ -24,9 +24,12 @@ If you have not signed up before, you can do so [here](https://improbable.io/get
 
 - Make sure you download the **Installer** version, and select the following components during installation:
     - **Linux Build Support**
-    - **Mac Build Support**
+    - **Mac Build Support** 
 
-> **Note:** You need Linux build support because all server-workers in a cloud deployment run in a Linux environment. 
+> **Note:** 
+You need these build supports even though you are developing on a Windows PC.<br/>
+You need **Linux Build Support** because all server-workers in a cloud deployment run in a Linux environment. <br/>
```
Comment (anne-edwards) [inline docs/get-started.md]: Minor point, but the capitalisation and bolding are inconsistent (elsewhere, only the OS is bold, and "Build Support" is lower case).

**Pull request merged**